.github/actions/javascript/contributorChecklist/contributorChecklist.js

```diff
@@ -137,16 +146,19 @@ getPullRequestBody()
         let authorChecklistComplete = false;
         let reviewerChecklistComplete = false;
 
-        // Once we've gathered all the data, loop through each comment and look to see if it contains a completed checklist
+        const authorChecklistNumber = countCheckedBoxes(completedAuthorChecklist);
+        const reviewerChecklistNumber = countCheckedBoxes(completedReviewerChecklist);
+
+        // Once we've gathered all the data, loop through each comment and look to see if it contains enough completed checkboxes
         for (let i = 0; i < combinedData.length; i++) {
-            const whitespace = /([\n\r])/gm;
-            const comment = combinedData[i].replace(whitespace, '');
+            const comment = combinedData[i];
+            const commentChecklistNumber = countCheckedBoxes(comment);
```

Comment:
Maybe we would still want the comment to differentiate from others somehow. I dont think it should be an issue to only check for the checkboxes since I dont think there is a high chance some random comment would have so many checkboxes 😄 

but just putting this out here.

Reply:
Maybe like if it has `PR Reviewer Checklist` or `PR Author Checklist` in the comment as well?

Reply:
Maybe I would not rely on that, I can imagine people would only copy the checkboxes, I think this is good.